Assets/Scripts/CameraFollow.cs

```diff
@@ -23,7 +25,9 @@ private void Start()
     {
         // Setting up the reference.
 		findPlayer();
```

Comment:
I think you can just add in the findPlayer() method to update the current camera maxSpeed, using the player's maxSpeed (that might have been slowed down)